vector/src/main/java/im/vector/app/features/createdirect/CreateDirectRoomActivity.kt

```diff
@@ -167,10 +166,10 @@ class CreateDirectRoomActivity : SimpleFragmentActivity() {
 
     private fun renderCreateAndInviteState(state: Async<String>) {
         when (state) {
-            Uninitialized,
```

Comment:
not a strong preference, I would be for keeping the case rather than relying on the else (has the benefit of catching areas that would change if another state was introduced)

```kotlin
when(state) {
  Uninitialized -> Unit
}
```

Reply:
I agree, but using `else` has the advantage to not change the format of the other cases (arrow alignment would have changed :/).

Also there is a few chance that another case is introduced to `Async`, this is pretty stable.

Reply:
Case and point against arrow alignment 👀